.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,54 +1,31 @@
----
 name: Bug report
 about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
 
----
-
 ### Bug Title:
 Please provide a CONCISE description of the bug.
 
----
 ### Bug Description:
 Please provide a DETAILED description of the bug.
 
----
 ### Reproduction Steps:    
 **1.**    
 **2.** If needed, delete if not    
 **3.** If needed, delete if not
 
----
 ### Expected Behavior:
 What should have happened if the bug didn't occur.
 
----
 ### Actual Behavior:
```

Comment:
I think actual behavior is very close to the bug description and could go as well.